### PR TITLE
Add `namespace` argument to Ray client get actor call

### LINKED ISSUE #17843: `namespace` is not valid argument for `get_actor` when using Ray client
### What is the problem?
`namespace` is not a valid argument for `get_actor` when using Ray client, but it is valid when using `ray.init()`

*Ray version and other system information (Python version, TensorFlow version, OS):* Python 3.7, Ray latest master `6957ce66f6eb370d633444b4420c874b2e517505`

### Reproduction (REQUIRED)
Please provide a short code snippet (less than 50 lines if possible) that can be copy-pasted to reproduce the issue. The snippet should have **no external library dependencies** (i.e., use fake or mock data / environments):

Start Ray with port: `ray start --head --ray-client-server-port 49152`
```
import ray

ray.util.connect("localhost:49152")

@ray.remote
class A:
	pass

A.options(name="a1", namespace="namespace1").remote()

ray.get_actor("a1", namespace="namespace1")
```
Error: `TypeError: get_actor() got an unexpected keyword argument 'namespace'`

If the code snippet cannot be run by itself, the issue will be closed with "needs-repro-script".

- [x] I have verified my script runs in a clean environment and reproduces the issue.
- [x] I have verified the issue also occurs with the [latest wheels](https://docs.ray.io/en/master/installation.html).

### PR DESCRIPTION
**Pull request by ericl**
Closes https://github.com/ray-project/ray/issues/17843

### CODE REVIEW

File: src/ray/protobuf/ray_client.proto [hunk 1]
```diff
@@ -61,6 +61,8 @@ message ClientTask {
   // A name parameter, if the payload can be called in more than one way
   // (like a method on a payload object).
   string name = 2;
+  // A namespace parameter.
+  string namespace = 9;
```
[the remark this answers is not in the document]
Comment (ijrsvt) [reply src/ray/protobuf/ray_client.proto:65]: WDYM by intended way? I think that putting it as index **9**  is fine. The order here I think is arbitrary, so putting it 3rd is fine.

**Review by ijrsvt — COMMENTED** [review]
Can you make sure to bump the protocol version? it is here https://github.com/ray-project/ray/blob/35d86ebfeee33d5415c9537d8e6214ad64332c5a/python/ray/util/client/__init__.py#L12

File: src/ray/protobuf/ray_client.proto (hunk 1 above)
[the remark this answers is not in the document]
Comment (ericl) [reply src/ray/protobuf/ray_client.proto:65]: I guess I was a bit confused by this proto, which seems overly generic in supporting multiple method calls. But I guess this is fine.

[the remark this answers is not in the document]
Comment (ijrsvt) [reply src/ray/protobuf/ray_client.proto:65]: Yeah, it is definitely a bit overly broad :(

File: python/ray/util/client/server/server.py [hunk 2]
```diff
@@ -477,7 +477,7 @@ def _schedule_named_actor(self,
                               task: ray_client_pb2.ClientTask,
                               context=None) -> ray_client_pb2.ClientTaskTicket:
         assert len(task.payload_id) == 0
-        actor = ray.get_actor(task.name)
+        actor = ray.get_actor(task.name, task.namespace or None)
```
Comment (ijrsvt) [inline python/ray/util/client/server/server.py:480]: nit: Can you mention that this `or` here is to convert an empty string -> `None`?

Comment (ericl) [reply python/ray/util/client/server/server.py:480]: Done

**Review by ijrsvt — APPROVED** [review]
LGTM now! Thanks @ericl

**Pull request merged**